ops/dockerfiles/server.Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM mycro/solc-python:latest
+
+#setup uwsgi used in prod environments
+RUN apt-get update
+RUN pip3 install --upgrade pip
```

Comment:
Is this needed? It adds layers to the image which makes it bigger.

Reply:
I don't know what the best practice is for including updates/upgrades to package managers. mycro/solc does one I believe but that can get cached for very long periods of time and it may not be clear to future developers that it's happening at all.

I view this as sort of like a defensive copy on both sides of a getX function. Has image size been an issue? If not I'd prefer to leave this I think.